style.css

```diff
@@ -1,7 +1,7 @@
 body {
```

Comment:
- [ ] [OPTIONAL]  Please pay attention that the social icons and the arrow below are not centered: 
This is happening because `ul` tag with `socials` class in header has padding, 
You can easily center it by setting `padding : 0`

With `padding:0 ` | Without `padding:0`
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/145389587-78376dc0-7322-40be-b4b0-76f8962fb95a.png)  | ![image](https://user-images.githubusercontent.com/77038610/145389536-0439464f-6519-47a0-b172-82532a36c96d.png)

It would also be nice if you set `align-items: center;`, because right won the M icon is jumping up 😁 
![image](https://user-images.githubusercontent.com/77038610/145390668-f21ef24c-6fcd-488a-b11b-3b99103555a7.png)

Reply:
Yeah. I see it thanks.